openmetadata-ui/src/main/resources/ui/src/components/ServiceConfig/ConnectionConfigForm.tsx

```diff
@@ -41,8 +42,8 @@ interface Props {
   serviceType: string;
   serviceCategory: ServiceCategory;
   status: LoadingState;
-  onFocus: (fieldName: string) => void;
-  onSave: (data: ISubmitEvent<ConfigData>) => void;
+  onFocus: (id: string) => void;
+  onSave: (data: IChangeEvent<ConfigData>) => void;
```

Comment:
Can you check if this is the correct type because here we are handling the onSubmit changes?

Reply:
Yes it's correct. https://github.com/rjsf-team/react-jsonschema-form/blob/91bcc94fcb2c73b8a06c85b4ed00c347843f228e/packages/playground/src/components/Playground.tsx#L124